apps/css-workshop/src/components/IconButton.astro

```diff
@@ -0,0 +1,29 @@
+---
+type Props = {
+  variant?: 'high-visibility' | 'borderless' | 'cta';
+  size?: 'small' | 'large';
+  isActive?: boolean;
+  shift?: 'left' | 'right';
+} & astroHTML.JSX.ButtonHTMLAttributes;
+
+const { class: className, variant, size, isActive, shift, ...props } = Astro.props;
+---
+
+<button
+  class:list={['iui-button', 'iui-button-base', className]}
+  type='button'
+  data-iui-variant={variant}
+  data-iui-size={size}
+  data-iui-active={isActive ? 'true' : undefined}
+  data-iui-shift={shift}
+  {...props}
+  ><slot name='button-icon' />
```

Comment:
This one should use the default slot instead of a named slot.

The usage of this component should look like this (notice the lack of extra attributes on the svg):

```jsx
<IconButton_ variant='borderless'>
  <svg-close-small></svg-close-small>
</IconButton_>
```

Reply:
Will remove.

Reply:
Done in this [commit](https://github.com/iTwin/iTwinUI/pull/2012/commits/2391a14be3c867e3f2d5f137ba6b53ea4feeef6d).

Reply:
You can remove `aria-hidden` from the various places where `IconButton_` is used, because it should already be handled within the main component.

Reply:
Okay, will remove.

Reply:
Removed in this [commit](https://github.com/iTwin/iTwinUI/pull/2012/commits/718becec0a4308c59c77764db16f0bb98ebde9e8).